sync-team/src/github/mod.rs

```diff
@@ -676,25 +676,33 @@ impl CreateRepoDiff {
 
 impl std::fmt::Display for CreateRepoDiff {
     fn fmt(&self, mut f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let CreateRepoDiff {
+            org,
+            name,
+            settings,
+            permissions,
+            branch_protections,
+        } = self;
+
         let RepoSettings {
             description,
             homepage,
             archived: _,
             auto_merge_enabled,
-        } = &self.settings;
+        } = &settings;
 
         writeln!(f, "➕ Creating repo:")?;
-        writeln!(f, "  Org: {}", self.org)?;
-        writeln!(f, "  Name: {}", self.name)?;
-        writeln!(f, "  Description: {:?}", description)?;
-        writeln!(f, "  Homepage: {:?}", homepage)?;
-        writeln!(f, "  Auto-merge: {}", auto_merge_enabled)?;
+        writeln!(f, "  Org: {org}")?;
+        writeln!(f, "  Name: {name}")?;
+        writeln!(f, "  Description: {description}")?;
```

Comment:
The description was originally `Option<String>`, which is why it was logged with `:?`. But recently we switched it to just `String`, so it doesn't make sense to print it as `Debug` anymore.